nautobot_golden_config/forms.py

```diff
@@ -330,18 +336,41 @@ class GoldenConfigSettingFeatureForm(
 ):
     """Filter Form for GoldenConfigSettingFeatureForm instances."""
 
+    backup_repository = forms.ModelMultipleChoiceField(
+        queryset=GitRepository.objects.filter(provided_contents__contains="nautobot_golden_config.backupconfigs"),
+        widget=StaticSelect2Multiple(),
+    )
+    intended_repository = forms.ModelMultipleChoiceField(
+        queryset=GitRepository.objects.filter(provided_contents__contains="nautobot_golden_config.intendedconfigs"),
+        widget=StaticSelect2Multiple(),
+    )
+
     class Meta:
         """Filter Form Meta Data for GoldenConfigSettingFeatureForm instances."""
 
         model = models.GoldenConfigSetting
         fields = (
             "backup_repository",
+            "backup_repository_template",
             "backup_path_template",
             "intended_repository",
+            "intended_repository_template",
             "intended_path_template",
             "jinja_repository",
             "jinja_path_template",
             "backup_test_connectivity",
             "scope",
             "sot_agg_query",
         )
+
+    def clean(self):
```

Comment:
I think this makes sense, but I think the logic needs to be applied to the model as well, as there is an api to the settings as well.